tests/migrations/test_questioner.py

```diff
@@ -35,20 +39,22 @@ class QuestionerHelperMethodsTests(SimpleTestCase):
 
     @mock.patch('builtins.input', return_value='datetime.timedelta(days=1)')
     def test_questioner_default_timedelta(self, mock_input):
-        questioner = InteractiveMigrationQuestioner()
-        with captured_stdout():
-            value = questioner._ask_default()
+        with captured_stdout() as stdout:
+            self.questioner.prompt_destination = stdout
```

Comment:
Do we need to use `captured_stdout()` anymore? I would initialize `questioner` with a custom prompt destination instead, e.g.:
```python
    def setUp(self):
        self.prompt = StringIO()
        self.questioner = InteractiveMigrationQuestioner(prompt_destination=self.prompt)

    @mock.patch('builtins.input', return_value='datetime.timedelta(days=1)')    
    def test_questioner_default_timedelta(self, mock_input):
        value = self.questioner._ask_default()
        self.assertEqual(value, datetime.timedelta(days=1))

    ...

    @mock.patch('builtins.input', side_effect=['bad code', 'exit'])
    def test_questioner_no_default_bad_user_entry_code(self, mock_input):
        with self.assertRaises(SystemExit):
            self.questioner._ask_default()
        self.assertIn('Invalid input: ', self.prompt.getvalue())
```
etc.